### PR TITLE
remove forced stubbed data and functionality to ingest data from data…

### PR DESCRIPTION
**Pull request by dvoong**
…workspace, added an environment variable to configure this

### CODE REVIEW

File: app/etl/tasks/core/__init__.py [hunk 1]
```diff
@@ -14,9 +14,9 @@
     extract_datahub_export_countries,
     extract_datahub_future_interest_countries,
     extract_datahub_interactions,
-    extract_datahub_omis_dataset,
-    extract_datahub_sectors,
-    extract_export_wins,
+    extract_datahub_omis,
+    # extract_datahub_sectors,
```
Comment (marcuspp) [inline app/etl/tasks/core/__init__.py]: Are these meant to be commented out?

File: app/etl/tasks/core/source_data_extraction.py [hunk 2]
```diff
@@ -2,108 +2,124 @@
...
+class SourceDataExtractor:
+    def __call__(self):
+        if current_app.config['app']['stub_source_data']:
+            return populate_table(self.stub_data, self.schema, self.table_name)
+        else:
+            dataworkspace_config = current_app.config['dataworkspace']
+            dataset_id = dataworkspace_config[self.dataset_id_config_key]
+            source_table_id = dataworkspace_config[self.source_table_id_config_key]
+            endpoint = f'api/v1/dataset/{dataset_id}/{source_table_id}'
+            url = 'http://{}/{}'.format(dataworkspace_config['host'], endpoint)
```
Comment (marcuspp) [inline app/etl/tasks/core/source_data_extraction.py]: For consistency (with the line above) use an f string.

File: app/etl/tasks/core/source_data_extraction.py [hunk 3]
```diff
@@ -182,55 +200,74 @@ def get_hawk_headers(
...
+    while next_page is not None:
+        response = requests.get(next_page, headers=headers)
         data = response.json()
+        output = populate_table(data, schema, table_name, overwrite=next_page == url)
+        n_rows = n_rows + output['rows']
+        next_page = data['next']
+    return {'table': table_name, 'rows': n_rows, 'status': 200}
+
+
+def populate_table(data, schema, table_name, overwrite=True):
```
Comment (marcuspp) [inline app/etl/tasks/core/source_data_extraction.py]: Does this need to be in a db transaction block?

File: tests/etl/test_populate_database.py [hunk 4]
```diff
@@ -9,9 +9,9 @@
 @patch('app.etl.tasks.core.extract_datahub_export_countries')
 @patch('app.etl.tasks.core.extract_datahub_future_interest_countries')
 @patch('app.etl.tasks.core.extract_datahub_interactions')
-@patch('app.etl.tasks.core.extract_datahub_omis_dataset')
-@patch('app.etl.tasks.core.extract_datahub_sectors')
-@patch('app.etl.tasks.core.extract_export_wins')
+@patch('app.etl.tasks.core.extract_datahub_omis')
+# @patch('app.etl.tasks.core.extract_datahub_sectors')
```
Comment (marcuspp) [inline tests/etl/test_populate_database.py]: Are you planning to re-add these in at a later date?

File: tests/etl/test_source_data_extraction.py [hunk 5]
```diff
@@ -0,0 +1,303 @@
...
+
+
+class SourceDataExtractBaseTestCase:
+
+    __test__ = False
+
+    @patch('app.etl.tasks.core.source_data_extraction.current_app')
+    def test_stub_data(self, current_app, app_with_db):
+        current_app.config = {'app': {'stub_source_data': True}}
+        # with app.app_context():
```
Comment (marcuspp) [inline tests/etl/test_source_data_extraction.py]: Remove

File: tests/etl/test_source_data_extraction.py [hunk 6]
```diff
@@ -0,0 +1,303 @@
...
+                ],
+                data,
+            )
+        )
+        assert data == self.expected_data
+
+
+class TestExtractDatahubCompany(SourceDataExtractBaseTestCase):
+
+    __test__ = True
```
Comment (marcuspp) [inline tests/etl/test_source_data_extraction.py]: Does this need to be here? Or does it inherit False from its parent?

File: utils/__init__.py [hunk 7]
```diff
@@ -0,0 +1,37 @@
+from flask import request
```
Comment (marcuspp) [inline utils/__init__.py]: Does these utils tests?

File: app/etl/tasks/core/__init__.py (hunk 1 above)
Comment (dvoong) [reply app/etl/tasks/core/__init__.py]: yeah these are post mvp items

File: tests/etl/test_populate_database.py (hunk 4 above)
Comment (dvoong) [reply tests/etl/test_populate_database.py]: yep

File: tests/etl/test_source_data_extraction.py (hunk 6 above)
Comment (dvoong) [reply tests/etl/test_source_data_extraction.py]: yeah it inherits it from its parent which is False

File: utils/__init__.py (hunk 7 above)
Comment (dvoong) [reply utils/__init__.py]: don't understand...

File: app/etl/tasks/core/source_data_extraction.py (hunk 3 above)
Comment (dvoong) [reply app/etl/tasks/core/source_data_extraction.py]: done

File: utils/__init__.py (hunk 7 above)
Comment (marcuspp) [reply utils/__init__.py]: Sorry, Are there any tests for the utils?

Comment (dvoong) [reply utils/__init__.py]: I believe these are covered by our integration tests

**Pull request merged**